apps/web/app/Nav.tsx

```diff
@@ -1,43 +1,33 @@
 "use client";
 
-import Link from "next/link";
 import styled from "styled-components";
-import { Logo } from "ui";
+import { Icon, Logo, Avatar } from "ui";
 
 const NavBar = styled.nav`
   box-sizing: border-box;
   display: flex;
   width: 100%;
   background-color: #1e4c40;
-  margin-bottom: 20px;
-  padding: 10px 20px 10px 40px;
+  margin:0;
+  padding: 15px 20px 15px 40px;
   justify-content: space-between;
+  z-index:1;
 `;
 
 const ActionWrapper = styled.div`
   display: flex;
   min-width: 150px;
-  justify-content: space-between;
-`;
-
-const LinkStyled = styled(Link)`
-  text-decoration: none;
-  color: #fff;
-  font-weight: bold;
-  font-size: 1.5em;
-
-  &:hover {
-    text-decoration: underline;
-  }
+  justify-content: space-around;
+  align-items: center;
 `;
 
 export default function Nav() {
   return (
     <NavBar>
       <Logo white />
       <ActionWrapper>
-        <LinkStyled href="/">Home</LinkStyled>{" "}
-        <LinkStyled href="/users">Users</LinkStyled>
+        <Icon icon={"notifications"} color={"white"}/>
+        <Avatar src="../public/avatar.png" outlined={true} />
```

Comment:
```suggestion
        <Avatar src="..avatar.png" outlined />
```
even! There is no need to pass true for a boolean attribute.